modules/minwinsvc/svc_windows.go

```diff
@@ -71,6 +71,4 @@ func (runner) Execute(args []string, r <-chan svc.ChangeRequest, changes chan<-
 			return false, 0
 		}
 	}
-
-	return false, 0
```

Comment:
I think this is needed because in the switch case in for loop, there is no default case so there is possibility that `return false, 0` might not happen so you may this here.

Reply:
As i see code is unreachable, so reason for leave it here, i'm also don't see.